test/valgrind/valgrindTestSuite.sh

```diff
@@ -590,7 +634,10 @@ then
 
     if [ "$lost" != "0" ]
     then
-      failedTest "test/valgrind/$vtest.*" $vtest $lost "valgrind" $xTestNo
+      leakFound "test/valgrind/$vtest.*" $vtest $lost "valgrind" $xTestNo
```

Comment:
Ah, this is bad.
Before there was only two options, now there are three.
This I will need to change